packages/swapper/src/swappers/cow/CowSwapper.test.ts

```diff
@@ -57,7 +57,7 @@ describe('CowSwapper', () => {
 
   describe('name', () => {
     it('returns the correct human readable swapper name', () => {
-      expect(swapper.name).toEqual('CowSwap')
+      expect(swapper.name).toEqual('CoW Swap')
```

Comment:
Can we use the `SwapperName` enum here?

Reply:
Ab-solutely - ae7e7726f5e1b3814dce80d4837fc022a8d4c22a 🧙🏽